### ISSUE TITLE
Railway level crossing triggers "bridge height" quest

### ISSUE THREAD

**Issue by derfasthirnlosenick** [streetcomplete/StreetComplete#5223]
Railway crossing w.o. bridge triggers quest asking for height of that bridge.
See https://www.openstreetmap.org/note/3858686

**How to Reproduce**
Probably triggers on most railway crossings?

**Expected Behavior**
Shouldn't trigger that quest if "level_crossing" is tagged.

**Versions affected**
54.0-beta1

**Comment by westnordost** [comment]
Looks like a bug to me, a cursory look at the tagging of the ways around it looks like everything is as it is supposed to be.

**Comment by matkoniecz** [comment]
Seems to be result of https://github.com/streetcomplete/StreetComplete/issues/5180 ?

**Comment by matkoniecz** [comment]
https://github.com/streetcomplete/StreetComplete/commit/50d6f7e477892e03ad44d5e9c49e8abb4423c0b7 has not provided separate message to be asked...

And changing it from asking about bridge to something less specific would it make quite confuisng

**Comment by derfasthirnlosenick** [comment]
> [50d6f7e](https://github.com/streetcomplete/StreetComplete/commit/50d6f7e477892e03ad44d5e9c49e8abb4423c0b7) has not provided separate message to be asked...
> 
> And changing it from asking about bridge to something less specific would it make quite confuisng

Yeah either that message would need to become "What's the height limit here?" or something similar, or add it as a separate quest with its own language.

**Comment by matkoniecz** [comment]
Or message can change depending on tagging situation.

**Comment by leijurv** [comment]
Reopen? I am still seeing "What height limit is indicated here? Railroad-Road Crossing" at nodes tagged with `railway=level_crossing`. 

There is no bridge here of any kind, so there is no height limit.

**Comment by matkoniecz** [comment]
> "What height limit is indicated here? Railroad-Road Crossing" at nodes tagged with railway=level_crossing.

this is intentional for electrified railways

> There is no bridge here of any kind, so there is no height limit.

catenary at least in some areas will have signed height limit

is it never signed in yours?

is railway tagged as having catenary and it does not have it?

**Comment by leijurv** [comment]
This is San Francisco around here https://www.openstreetmap.org/#map=17/37.762918/-122.467690 and I think it is never signed. Google says the lines are significantly higher than the tallest legal vehicle (including trucks).
